### PR TITLE
mon: fix hang on deprecated/removed 'pg set_*full_ratio' commands

### PR DESCRIPTION
**Pull request by liewegas**
Fixes: http://tracker.ceph.com/issues/20600

### CODE REVIEW

**Comment by tchaikov** [comment]
retest this please.

File: src/mon/PGMonitor.cc [hunk 1]
```diff
@@ -558,21 +558,26 @@ version_t PGMonitor::get_trim_to()
...
+    }
     return preprocess_pg_stats(op);
 
   case MSG_MON_COMMAND:
+    if (mon->osdmon()->osdmap.require_osd_release >= CEPH_RELEASE_LUMINOUS) {
+      bufferlist rdata;
+      mon->reply_command(op, -EOPNOTSUPP, "this command is obsolete", rdata,
+			 get_last_committed());
+      return false;
+    }
```
Comment (gregsfortytwo) [inline src/mon/PGMonitor.cc]: Shouldn't these return true, as they've been handled?

Comment (liewegas) [inline src/mon/PGMonitor.cc]: yep, updated!

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-07-13_20:38:15-rados-wip-sage-testing-distro-basic-smithi/

**Pull request merged**